test/rebar_test_utils.erl

```diff
@@ -105,12 +105,12 @@ create_config(AppDir, Contents) ->
 
 %% @doc Util to create a random variation of a given name.
 create_random_name(Name) ->
-    random:seed(os:timestamp()),
+    random:seed(erlang:now()),
```

Comment:
As I mentioned above, using `os:timestamp/0` causes rebar_ct_SUITE:multi_app_default_dir/1 on windows to fail.